internal/expr/parser_helper_test.go

```diff
@@ -132,3 +132,47 @@ func (s *ExprInternalSuite) TestUnfinishedQuote(c *C) {
 		}
 	}
 }
+
+func (s *ExprInternalSuite) TestValidComments(c *C) {
+	var p = NewParser()
+
+	validComments := []string{
+		`-- Simple comment`,
+		`-- Simple with line break
+		`,
+		`/* muti
+		 line */`,
+		`/* unfinished mutiline`,
+		`/* -- */`,
+		`-- */`,
+	}
+
+	for _, q := range validComments {
+		p.init(q)
+		ok := p.skipComment()
+		if !ok {
+			c.Errorf("test failed. %s is a valid comment", q)
```

Comment:
maybe this could be a bit clearer message, what do you think?:
"test failed. Test outcome for valid comment %q is found to be invalid", q

Reply:
Made a bit clearer.